dev/com.ibm.websphere.appserver.org.eclipse.microprofile.opentracing-1.0/com.ibm.websphere.appserver.org.eclipse.microprofile.opentracing-1.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.opentracing-1.0
+visibility=public
+IBM-ShortName: opentracing-api-1.0
```

Comment:
Should this be a public feature at all? I thought this is the api private feature and it is a singleton as well.